runtime/mux.go

```diff
@@ -89,6 +94,20 @@ func WithUnescapingMode(mode UnescapingMode) ServeMuxOption {
 	}
 }
 
+// WithMiddlewares sets server middleware for all handlers
+func WithMiddlewares(middlewares ...Middleware) ServeMuxOption {
+	return func(serveMux *ServeMux) {
+		serveMux.middlewares = append(serveMux.middlewares, middlewares...)
+	}
+}
+
+// WithInjectHTTPPattern sets the current HTTP Pattern in the request context
+func WithInjectHTTPPattern() ServeMuxOption {
+	return func(serveMux *ServeMux) {
+		serveMux.injectHTTPPattern = true
+	}
+}
```

Comment:
I don't think we need an option for this, is there any downside to doing this unconditionally in the handler wrapper?

Reply:
Nothing critical will happen if the pattern is added all the time. But it seemed to me that when updating the library, users would be concerned that there would be more contexts, so I thought it would be nice to be able to control this

Reply:
Nah, remove it.

Reply:
@johanbrandhorst done